scripts/test_images.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+IFS=$'\n\t'
+
+source ./scripts/test_lib.sh
+source ./scripts/build_lib.sh
+
+function startContainer {
+    # run docker in the background
+    docker run -d --rm --name "${RUN_NAME}" "${IMAGE}"
+
+    # wait for etcd daemon to bootstrap
+    sleep 5
+}
+
+function runVersionCheck {
+    Out=$(docker exec "${RUN_NAME}" "${@}")
```

Comment:
Any reason that we use `docker exec` here instead of `docker run --rm`?
I would allow make testing version independent from testing etcdctl get